sgtm/artemis/README.md

```diff
@@ -0,0 +1,259 @@
+# Artemis
```

Comment:
Can you help me understand how this differs from the built-in Firestore read variable, [see here](https://www.simoahava.com/analytics/enrich-server-side-data-with-cloud-firestore/).

Can you make this difference clear in the docs, as others might wonder the same thing

Reply:
Yep - added this to the readme:

## Why use this tag when sGTM has a Firestore lookup feature already?

sGTM has a variable type which allows you lookup data in Firestore. The key
difference between that and Artemis is that this built-in lookup gets data from
an attribute within a document whereas Artemis allows you to get the entire 
document and parse out any value. This allows you to return more data in a 
single API call, reducing costs and improving performance.

If you only require a single value to be looked up, the built-in lookup would
be best to use.

Reply:
That's great, and explains well.

Sorry if I'm being daft, but have you added that already, or you will in the next commit?